### PR TITLE
Add replit login loop script

### PR DESCRIPTION
**Pull request by bhackett1024**

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-let browserName = process.env.PLAYWRIGHT_CHROMIUM ? "chromium" : "firefox";
+process.env.RECORD_ALL_CONTENT = 1;
+
+let browserName = "chromium";
 const launchOptions = {
   headless: !!process.env.PLAYWRIGHT_HEADLESS,
-  ...devices[browserName === "firefox" ? "Replay Firefox" : "Replay Chromium"]
-    .launchOptions,
+  executablePath: process.env.HOME + "/tmp/Replay-Chromium.app/Contents/MacOS/Chromium",
```
Comment (Domiii) [inline src/helpers.js:10]: 1. You don't need to create your own record/replay script. We already have that in `test/runtime/runTest.ts` ([src](https://github.com/replayio/backend/blob/4fc202a9063dd17dbcdd171ee8b83b668f995b61/test/runtime/runTest.ts)).
2. Using `RECORD_REPLAY_DIRECTORY` and `REPLAY_CHROMIUM_EXECUTABLE_PATH` should be sufficient. If it is not, we should fix that.

File: examples/replit-login-loop.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+const { example } = require("../src/helpers");
+
+const selectors = {
```
Comment (Domiii) [inline examples/replit-login-loop.js:3]: No need to do this manually. You can create Playwright tests with codegen. I have added [this video](https://www.youtube.com/watch?v=LM4yqrOzmFE) on how it works to [TT-1090](https://linear.app/replay/issue/TT-1090/crash-fix-automate-the-record-then-replay-until-it-crashes-loop).

**Review by Domiii — APPROVED** [review]
Approving so you are not held back. But, ideally, this should not be necessary.

**Comment by bhackett1024** [comment]
Thanks but I wasn't really looking for a review for this, a lot of this code is copy pasted from another script in the repo and it isn't intended for landing, I should have marked this a draft.

File: examples/replit-login-loop.js (hunk 2 above)
Comment (Domiii) [reply examples/replit-login-loop.js:3]: Update: I see what you did. I guess, you copy-and-pasted the QA Wolf script 👀

**Pull request closed without merging**